clients/admin-ui/src/features/common/modals/FilterModal.tsx

```diff
@@ -47,7 +47,12 @@ export const FilterModal: React.FC<FilterModalProps> = ({
       <ModalHeader>Filters</ModalHeader>
       <ModalCloseButton />
       <Divider />
-      <ModalBody maxH="85vh" padding="0px" overflowX="auto">
+      <ModalBody
+        maxH="85vh"
+        padding="0px"
+        overflowX="auto"
+        style={{ scrollbarGutter: "stable" }}
```

Comment:
This prevents the flicker from the scrollbar popping in when the contents overflow for the first time